src/main/java/org/folio/dew/batch/eholdings/EHoldingsToExportFormatMapper.java

```diff
@@ -179,6 +181,11 @@ private String convertBoolToStr(Boolean isTrue) {
     return Boolean.TRUE.equals(isTrue) ? "Yes" : "No";
   }
 
+  private String mapToken(Token token) {
+    if (isNull(token)) { return ""; }
+    return token.getPrompt() + ':' + token.getValue();
```

Comment:
There should be another separator according to requirements:
![image](https://user-images.githubusercontent.com/5904625/192506576-b69618e8-1659-44c9-b071-d32fa06100db.png)